MANIFESTO.md

```diff
@@ -1 +1,6 @@
 
+##Community
+
+Mankind did not evolve by sitting alone in the dark. For the software industry to really grow in the Caribbean we need collaboration, knowledge and skill sharing. Avoid platforms and tools that do not respect communities that form around them or worse; shun them.
+
+Community does not only mean software developers, after all we need to solve somebody's problems. Developers should therefore take time to interact with and integrate with the rest of society.
```

Comment:
>If you disagree then explain why you don't think Community is necessary.

I didn't express a point of view in my feedback. I was merely asking you to explain why you believe what you believe.